.github/workflows/smoke-test.yml

```diff
@@ -12,12 +12,12 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 14
+          node-version: 16
```

Comment:
This is required in order to have `npm link` working between the workflow and the action.

Having 14, or even 12, here leads to `Failed to load plugin 'unicorn' declared in 'CLIOptions': Cannot find module 'eslint-plugin-unicorn'` error. The action is run as  [Github Docker container action](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action) using [`node:14-alpine` image](https://github.com/AriPerkkio/eslint-remote-tester-run-action/blob/master/Dockerfile#L1) so I have no idea why `node-version: 14` doesn't work here.